Ai Pin/API/API.swift

```diff
@@ -227,7 +227,7 @@ extension API {
         try await post(url: Self.noteUrl.appending(path: "create"), body: note)
     }
     
-    func update(id: String, with note: Note) async throws -> Bool {
+    func update(id: String, with note: Note) async throws -> Any {
```

Comment:
No, this is not correct. While the existing return type is wrong, it doesn't help to make it more wrong. The correct return type should be inspected and lined up with the appropriate model. In this case it is probably the`Memory` type.